src/pages/main.css

```diff
@@ -54,7 +54,11 @@ code span {
 
 /* Inline code */
 :not(pre) > code {
-  @apply px-2 py-1 rounded font-mono text-sm text-gray-800 bg-gray-100;
+  padding: 0.25em 0.5em;
+  border-radius: 0.25em;
+  font-size: 0.875em;
+  background: rgb(243, 244, 246);
+  color: rgb(31, 41, 55);
```

Comment:
Tailwind is adding excess letter spacing via `@apply`. Is this configurable via the typography plugin?